frontend/src/libs/helpers/get-deep-error-message.helper.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  type FormFieldErrors,
+  type FormFieldValues,
+  type FormPath,
+} from '#libs/types/types.js';
+
+const getDeepErrorMessage = <T extends FormFieldValues>(
```

Comment:
it's not a reusable helper because we may have a deeper nesting, so let's concat already defined fields in file input, without this helper